src/dash-table/components/CellInput/index.tsx

```diff
@@ -1,30 +1,33 @@
 import React, {
-    PureComponent,
-    KeyboardEvent
+    ChangeEvent,
+    ClipboardEvent,
+    KeyboardEvent,
+    MouseEvent,
+    PureComponent
 } from 'react';
-import Dropdown from 'react-select';
-
-import DOM from 'core/browser/DOM';
-
-import {
-    ICellDefaultProps,
-    ICellProps,
-    ICellPropsWithDefaults,
-    ICellState
-} from 'dash-table/components/CellInput/props';
 
 import {
     KEY_CODES, isNavKey
 } from 'dash-table/utils/unicode';
-import { ColumnType } from 'dash-table/components/Table/props';
-import dropdownHelper from 'dash-table/components/dropdownHelper';
 
-export default class CellInput extends PureComponent<ICellProps, ICellState> {
+interface ICellProps {
+    active: boolean;
+    className: string;
+    focused: boolean;
+    onChange: (e: ChangeEvent) => void;
+    onClick: (e: MouseEvent) => void;
+    onDoubleClick: (e: MouseEvent) => void;
+    onMouseUp: (e: MouseEvent) => void;
+    onPaste: (e: ClipboardEvent<Element>) => void;
+    type?: string;
+    value: any;
+}
+
+interface ICellState {
+    value: any;
+}
 
-    public static defaultProps: ICellDefaultProps = {
-        conditionalDropdowns: [],
-        type: ColumnType.Text
-    };
+export default class CellInput extends PureComponent<ICellProps, ICellState> {
```

Comment:
Standalone Input component -- what's left after removing Label and Dropdown from the existing implementation